packages/ERTP/src/types.d.ts

```diff
@@ -0,0 +1,347 @@
+import type { CopySet } from '@endo/patterns';
+
+export type NatValue = bigint;
+export type SetValue = any[];
+
+/**
+ * Amounts are descriptions of digital assets, answering the questions "how
+ * much" and "of what kind". Amounts are values labeled with a brand. AmountMath
+ * executes the logic of how amounts are changed when digital assets are merged,
+ * separated, or otherwise manipulated. For example, a deposit of 2 bucks into a
+ * purse that already has 3 bucks gives a new purse balance of 5 bucks. An empty
+ * purse has 0 bucks. AmountMath relies heavily on polymorphic MathHelpers,
+ * which manipulate the unbranded portion.
+ */
+export type Amount<K extends AssetKind = AssetKind> = {
+  brand: Brand<K>;
+  value: AssetValueForKind<K>;
+};
+/**
```

Comment:
Is the lack of blank lines between def'ns intentional?

```suggestion
};

/**
```

Reply:
it's what the type emitter built. I think it's okay.